qbft/spectest/tests/messages/unmarshal_justification.go

```diff
@@ -0,0 +1,38 @@
+package messages
+
+import (
+	"encoding/hex"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// UnmarshalJustifications tests unmarshalling justifications
+func UnmarshalJustifications() *tests.MsgSpecTest {
+	ks := testingutils.Testing4SharesSet()
```

Comment:
if this test fails, what will the failure/error look like?

Reply:
likely `incorrect offset` or something like
```
        	Error:      	Received unexpected error:
        	            	invalid ssz encoding. first variable element offset indexes into fixed value data
```